hack/test-cmd.sh

```diff
@@ -301,8 +301,10 @@ os::cmd::expect_success 'oc logout'
 os::cmd::expect_failure_and_text 'oc get pods' '"system:anonymous" cannot list pods'
 
 # make sure we report an error if the config file we pass is not writable
-# Does not work inside of a container, determine why and reenable
-# os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test '--config=${templocation}/file' --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
+templocation="/tmp/testconfig"
```

Comment:
Use something underneath `$BASETMPDIR` instead of something directly under `/tmp`.